src/cement_ai_platform/data/processors/data_prep_train_test_split_c1b218f5-2bc4-406e-ae2b-5523fcaea787.py

```
@@ -0,0 +1,42 @@
+import pandas as pd
+import numpy as np
+from sklearn.model_selection import train_test_split
+
+print("=== TRAIN/TEST SPLIT ===")
+
+# Prepare features and target
+# Use scaled numerical features and encoded categorical features for modeling
+model_features = numerical_features + ['category_A_encoded', 'category_B_encoded']
```

Ruff (0.13.1)

Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)